### ISSUE TITLE
Revisit SpecLength

### ISSUE THREAD

**Issue by samm82** [JacquesCarette/Drasil#1717]
As mentioned in [this comment](https://github.com/JacquesCarette/Drasil/commit/f04cafcea874e4422c861085aa34ba9e60db0fdc#r34291018), `specLength` should be revisited to not have a catch-all clause.

**Comment by samm82** [comment]
@JacquesCarette Currently, then the `specLength` of a `Ref` is 0, this is what the generated trace graph looks like:
![image](https://user-images.githubusercontent.com/35857611/61317732-9ddcaf80-a7d1-11e9-9c03-55e59cf02335.png)

When I change it be the following (which could very well be wrong), this is the generated graph:
```haskell
specLength x@Ref{}   = length $ filter (`notElem` dontCount) $ TP.render $ runPrint (spec x) Text
```
![image](https://user-images.githubusercontent.com/35857611/61317599-4d655200-a7d1-11e9-8495-efad68730f49.png)

It looks like in the second image, the headings are shifted to match the first column width of the first image (which I'm not sure why), but the "X"s aren't shifted.

**Comment by JacquesCarette** [comment]
There is indeed something terribly wrong with that output!  Of course, that count is also quite wrong [you should never render things and then post-process that output].  So you should fix the `specLength` to just count the characters of the label, and see what that does.

**Comment by samm82** [comment]
I'm having two issues now that I'll get to tackling later:
1) The headers of SSP are too big
2) The first column of the table(s) in SWHS aren't actually using the `longtabu`

My hunch is that these are the same issue, and I think the problem might be that there is an upper limit to how long a cell can be when using `longtabu`. I'll look into it though

**Comment by samm82** [comment]
The specific issue here is still unknown to me, but the underlying issue is that I defined the `specLength` of the `Ref`s almost completely wrong - it has been fixed in 04f276f17.
